### PR TITLE
Testing branch 1

### PR DESCRIPTION
**Pull request by NSMcEnroe**
## Description
UPDATE: Added the facade test :)

This PR will include tests for the services and poros methods.  Facade creation is built into the service, so as per Chris' talk yesterday, I decided against adding specific test creations for the facade.  Also, I couldn't find any good resources on that specific idea.

I also added a sad path if the user does not enter anything into the search bar, because before, it would have an empty results page.  I'm not finding too many sad paths or edge cases at the moment without utilizing the within blocks.

I'm going to take my lunch now, and hop back on afterwards.

### Type of Change

- [ ] **fix**
- [ ] **feat**
- [x] **test**
- [ ] **refactor**
- [ ] **docs**

### Checklist

- [x] **Code has been self reviewed**
- [x] **Code runs without any errors**
- [x] **Thorough testing has been implemented if adding feature**
- [x] **All tests pass**

### CODE REVIEW

File: spec/facades/movie_facades.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        movie_facade = SearchFacade.new
+
+        movies = movie_facade.movies
+
+        expect(movies).to be_an(Array)
+        expect(movies.first).to be_a(Movie)
+    end
+    end
+  end
+en
```
Comment (ttakahashi1591) [inline spec/facades/movie_facades.rb:15]: This testing for the movie facade looks good!

File: spec/services/movie_database_service_spec.rb [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        expect(movie_data).to have_key :original_title
+        expect(movie_data[:original_title]).to be_a(String)
+
+        expect(movie_data).to have_key :vote_average
+        expect(movie_data[:vote_average]).to be_a(Float)
+      end
+    end
+  
+  end
+end
```
Comment (ttakahashi1591) [inline spec/services/movie_database_service_spec.rb:38]: This services test looks good!

File: spec/services/movie_database_service_spec.rb [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+#         expect(member_data[:role]).to be_a(String)
+
+#         expect(member_data).to have_key :district
+#         expect(member_data[:district]).to be_a(String)
+
+#         expect(member_data).to have_key :party
+#         expect(member_data[:party]).to be_a(String)
+#       end
+#     end
+#   end
```
Comment (ttakahashi1591) [inline spec/services/movie_database_service_spec.rb:64]: We could probably remove this example from the test.
